### DISCUSSION TITLE
How does the SearXNG tool work?

### DISCUSSION THREAD

**Discussion by Fabix84** [FlowiseAI/Flowise#2989] [Q&A]
I have tried everything but I can't get the web search to work using the **SearXNG** tool. I don't have SearXNG installed locally but I'm trying to use the available public instances. However, whatever instance I try to insert into the base url I always get this error message. Has anyone managed to get this tool to work? Could you help me?

![Screenshot 2024-08-10 021307](https://github.com/user-attachments/assets/05d0378c-4d67-4c27-a56a-f39cd33088f7)

**Comment by dannykorpan** [comment]
Same error and question here... any advice?

**Comment by mdlmarkham** [comment]
Have you tried running SearXNG in Docker locally? That works well for me.

**Comment by dannykorpan** [comment]
I also get this error and now waiting for an updated docker built 😝
https://github.com/FlowiseAI/Flowise/issues/2815

<img width="669" alt="grafik" src="https://github.com/user-attachments/assets/14b97372-0725-4fd5-91e4-b409780563b7">
